### PR TITLE
10 2 x boosted

### PR DESCRIPTION
**Pull request by dmajumder**
extra variables for deepdublex. script for crab job submission.

### CODE REVIEW

File: python/defaults/Prompt18.py [hunk 1]
```diff
@@ -2,16 +2,18 @@
 	'eras' : ['Run2_2018'],
 	'miniAOD' : True,
 	'runDeepFlavourTagVariables' : True,
+        'doBoostedCommissioning': True,
+        'groups': ['EventInfo', 'Devdatta', 'DoubleBCommissioning'],
```
Comment (rauser) [inline python/defaults/Prompt18.py]: Please remove the above two lines, as the changes would apply for everyone

File: test/runBTagAnalyzer_cfg.py [hunk 2]
```diff
@@ -338,7 +338,7 @@
 )
 
 ## 'maxEvents' is already registered by the Framework, changing default value
-options.setDefault('maxEvents', -1)
+options.setDefault('maxEvents', 100)
```
Comment (rauser) [inline test/runBTagAnalyzer_cfg.py]: Please change back to -1, this affects all crab jobs of other people

File: test/runBTagAnalyzer_cfg.py [hunk 3]
```diff
@@ -584,18 +584,18 @@
...
+#  , 'pfDeepFlavourPrunedJetTags:problepb'
+#  , 'pfDeepFlavourPrunedJetTags:probc'
+#  , 'pfDeepFlavourPrunedJetTags:probuds'
+#  , 'pfDeepFlavourPrunedJetTags:probg'
+#  , 'pfNegativeDeepFlavourPrunedJetTags:probb'
+#  , 'pfNegativeDeepFlavourPrunedJetTags:probbb'
+#  , 'pfNegativeDeepFlavourPrunedJetTags:problepb'
+#  , 'pfNegativeDeepFlavourPrunedJetTags:probc'
+#  , 'pfNegativeDeepFlavourPrunedJetTags:probuds'
+#  , 'pfNegativeDeepFlavourPrunedJetTags:probg'
```
Comment (rauser) [inline test/runBTagAnalyzer_cfg.py]: What is the reason for commenting these taggers?

File: test/run_tests.sh [hunk 4]
```diff
@@ -3,6 +3,9 @@
 set -o errexit
 function die { echo 'FAILED': status $1 ; }
 
+echo 'Running on MINIAODSIM boosted'
+cmsRun runBTagAnalyzer_cfg.py defaults=Moriond19Boosted runOnData=False maxEvents=1000 &> boosted.log || die $?
```
Comment (rauser) [inline test/run_tests.sh]: Please reduce the number of test events to 20

**Comment by rauser** [comment]
@dmajumder thanks a lot for the PR which in general looks good to me. I have some small comments, please see above. Best, Johann

File: test/runBTagAnalyzer_cfg.py (hunk 3 above)
Comment (dmajumder) [reply test/runBTagAnalyzer_cfg.py]: @alefisico

Comment (alefisico) [reply test/runBTagAnalyzer_cfg.py]: no reason, sorry it must be leftover from a test

Comment (dmajumder) [reply test/runBTagAnalyzer_cfg.py]: @alefisico no problem I can fix it along with the other items.
Does this affect the ntuples we started making?

Comment (alefisico) [reply test/runBTagAnalyzer_cfg.py]: no because we dont use deepFlavour

**Comment by rauser** [comment]
@dmajumder Thanks for the fast implementation of the comments! In case there are no more comments by @alefisico I'll go ahead and merge.

**Comment by alefisico** [comment]
No more comments from me, thanks @rauser

**Review by alefisico — COMMENTED** [review]
Looks good to me :D

**Pull request merged**